client/app/bundles/course/assessment/assessments-edit.jsx

```diff
@@ -4,27 +4,27 @@ import ProviderWrapper from 'lib/components/ProviderWrapper';
 import storeCreator from './store';
 import EditPage from './pages/EditPage';
 
-const mountNode = document.getElementById('assessment-edit');
-if (mountNode) {
-  const dataAttr = mountNode.getAttribute('data');
-  const data = JSON.parse(dataAttr);
-  const store = storeCreator({});
+$(document).ready(() => {
+  const mountNode = document.getElementById('assessment-edit');
```

Comment:
because when the document is not ready, the mountNode is not there yet.

Actually I realized that our rails side JS has some magic and all JavaScripts are executed after the document is ready, so the code works before.